src/content/en/fundamentals/getting-started/primers/media-source-extensions.md

```diff
@@ -284,6 +289,7 @@ function sourceOpen(e) {
     .then(function(arrayBuffer) {
       <strong>sourceBuffer.onupdateend = function(e) {
         mediaSource.endOfStream();
+        window.URL.revokeObjectURL(vidElement.src);
```

Comment:
For consistency, let's use `URL.revokeObjectURL` instead of `window.URL.revokeObjectURL`